lib/file/file_windows.go

```diff
@@ -101,3 +112,24 @@ func IsReserved(path string) error {
 	}
 	return nil
 }
+
+// IsLongPathsEnabled checks if the current have the LongPathsEnabled enabled
+func IsLongPathsEnabled() bool {
```

Comment:
I conjecture that reading stuff from the registry might be time consuming. Rclone creates a lot of directories so should we cache this?

I'd suggest a couple of package level variables a [sync.Once](https://pkg.go.dev/sync#Once) and a bool to hold the cached value.